components/espressif__esp_secure_cert_mgr/.component_hash

```diff
@@ -0,0 +1 @@
+0a7b7f1c26fbd6fd8f49cc90f39203e3ab9cf334bbbf288bede9db745c35892c
```

Comment:
Any reason to commit the [ESP Secure Certificate Manager](https://components.espressif.com/components/espressif/esp_secure_cert_mgr) files? The IDF component manager is designed to download the files to each project specifically.